internal/service/dhcp/dhcpfailover_resource_test.go

```diff
@@ -0,0 +1,1230 @@
+package dhcp_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+
+	"github.com/infobloxopen/infoblox-nios-go-client/dhcp"
+	"github.com/infobloxopen/terraform-provider-nios/internal/acctest"
+	"github.com/infobloxopen/terraform-provider-nios/internal/utils"
+)
+
+var readableAttributesForDhcpfailover = "association_type,comment,extattrs,failover_port,load_balance_split,max_client_lead_time,max_load_balance_delay,max_response_delay,max_unacked_updates,ms_association_mode,ms_enable_authentication,ms_enable_switchover_interval,ms_failover_mode,ms_failover_partner,ms_hotstandby_partner_role,ms_is_conflict,ms_previous_state,ms_server,ms_state,ms_switchover_interval,name,primary,primary_server_type,primary_state,recycle_leases,secondary,secondary_server_type,secondary_state,use_failover_port,use_ms_switchover_interval,use_recycle_leases"
+
+func TestAccDhcpfailoverResource_basic(t *testing.T) {
+	var resourceName = "nios_dhcp_dhcpfailover.test"
+	var v dhcp.Dhcpfailover
+
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
```suggestion
	resource.Test(t, resource.TestCase{
```
As only one failover association between the same two servers is allowed, Please use resource.Test for all the test cases as they cannot run in parallel